### PR TITLE
Activities controller, model and migration added

### PR DESCRIPTION
**Pull request by bluemountains88**

### CODE REVIEW

File: migrations/20220715140317-create-activities.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        allowNull: false
+      },
+      createdAt: {
+        allowNull: false,
+        type: Sequelize.DATE
+      },
+      updatedAt: {
+        allowNull: false,
+        type: Sequelize.DATE
+      }
```
Comment (maxxCeballos) [inline migrations/20220715140317-create-activities.js:34]: falta atributo deletedAt

File: models/activities.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    static associate(models) {
+      // define association here
+    }
+  }
+  Activities.init({
+    id:{ 
+      type: DataTypes.INTEGER,
+      allowNull: false,
+      autoIncrement: true,
+    },
```
Comment (maxxCeballos) [inline models/activities.js]: Es necesario agregar el id?

**Review by maxxCeballos — CHANGES_REQUESTED** [review]

File: migrations/20220715140317-create-activities.js (hunk 1 above)
Comment (bluemountains88) [reply migrations/20220715140317-create-activities.js:34]: Atributo agreagado

File: models/activities.js (hunk 2 above)
Comment (bluemountains88) [reply models/activities.js]: Quite el id del modelo

**Review by maxxCeballos — APPROVED** [review]

**Pull request merged**